Rmarkdown/Multiple_Ascending_Dose_PK_KeyPlots.Rmd

```diff
@@ -24,7 +23,7 @@ Data specifications can be accessed on [Datasets](Datasets.html) and Rmarkdown t
   library(tidyr)
   library(dplyr)
   library(stringr)
-  library(xgxr, lib.loc = "../Rlib")
+  library(xgxr)#, lib.loc = "../Rlib")
```

Comment:
Why is there "#"? Was this to use the v1.0.7? Should we just install v1.0.7 in Rlib now?